content/manuals/desktop/vmm.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Virtual Machine Manager for Docker Desktop on Mac
+linkTitle: Virtual Machine Manager 
+params:
+  sidebar:
+    badge:
+      color: green
+      text: New
+keywords: virtualization software, resource allocation, mac, docker desktop, vm monitoring, vm performance, apple silicon
+description: Discover Docker Desktop for Mac's Virtual Machine Manager (VMM) options, including the new Docker VMM for Apple Silicon, offering enhanced performance and efficiency
+weight: 150
+---
+
+The Virtual Machine Manager (VMM) in Docker Desktop for Mac is responsible for creating and managing the virtual machine used to run containers. Depending on your system architecture and performance needs, you can choose from multiple VMM options in Docker Desktop's [settings](/manuals/desktop/settings.md#general). This page provides an overview of the available options.
+
+## Docker VMM (Beta)
+
+Docker VMM is a new, container-optimized hypervisor introduced in Docker Desktop 4.35 and available on Apple Silicon Macs only. It has been designed to enhance speed and resource efficiency, making it an ideal choice for developers seeking to optimize their workflow efficiency. 
+
+Docker VMM brings exciting advancements specifically tailored for Apple Silicon machines. By optimizing both the Linux kernel and hypervisor layers, Docker VMM delivers significant performance enhancements across common developer tasks. 
+
+Some key performance enhancements provided by Docker VMM include:
+ - Faster I/O operations: With a cold cache, iterating over a large shared filesystem with `find` is 2x faster than when the Apple Virtualization Framework is used.
+ - Improved caching: With a warm cache, performance can improve by as much as 25x, even surpassing native Mac operations.
+
+These improvements directly impact developers who rely on frequent file access, network interactions, and overall system responsiveness during containerized development. Docker VMM marks a significant leap in speed, enabling smoother workflows and faster iteration cycles.
```

Comment:
This paragraph talks about enhancements to "network interactions", but the key performance enhancements earlier seem to focus on faster I/O and filesystem caching. How can we back up this claim that networking is better?